src/pages/[boardId]/[paperId]/page.tsx

```
@@ -0,0 +1,36 @@
+import Header from '@/components/Header/Header';
+import {VStack} from '@/components/Stack';
+import Top from '@/components/Top/Top';
+import {Button} from '@/components/Button';
```

ESLint

[error] `@/components/Button` import should occur before import of `@/components/Header/Header`

(import/order)